documentation/src/main/asciidoc/quickstart/guides/obtaining.adoc

```diff
@@ -14,6 +14,7 @@ hibernate-osgi:: Hibernate support for running in OSGi containers.
 hibernate-c3p0:: Integrates the link:$$http://www.mchange.com/projects/c3p0/$$[C3P0] connection pooling library into Hibernate
 hibernate-hikaricp:: Integrates the link:$$http://brettwooldridge.github.io/HikariCP/$$[HikariCP] connection pooling library into Hibernate
 hibernate-proxool:: Integrates the link:$$http://proxool.sourceforge.net/$$[Proxool] connection pooling library into Hibernate
+hibernate-jcache:: Integrates the link:$$https://jcp.org/en/jsr/detail?id=107$$[JCache] caching specification into Hibernate as a second-level cache provider.
```

Comment:
I think I'd like to see this state more explicitly that hibernate-jcache is meant to integrate any JCache compliant caching library into Hibernate as a second level via the JCache APIs.